src/lib/feature-finder.js

```diff
@@ -22,10 +22,24 @@ export default function(cucumberOptions) {
       let featureData = parseFeature(file);
       return featureData.children
         .filter((child) => {
-          return (child.type === "Scenario" || child.type === "ScenarioOutline")
+          return (child.type === 'Scenario' || child.type === 'ScenarioOutline')
                  && verifyTags(child, cucumberOptions.tags);
         })
         .map((scenario) => {
+/*eslint-disable max-nested-callbacks*/
```

Comment:
Some of this could probably be extracted to a named function. Might improve clarity.